frontend/packages/ceph-storage-plugin/console-extensions.json

```diff
@@ -152,40 +152,40 @@
   {
     "type": "console.page/route",
     "properties": {
-      "path": "/odf/resource/:resourceKind/:resourceName",
+      "path": "/odf/resource/noobaa.io~v1alpha1~BackingStore/create/~new",
       "exact": true,
       "component": {
-        "$codeRef": "resourceDetailsPage.GenericDetailsPage"
+        "$codeRef": "bsCreate.default"
       }
     }
   },
   {
     "type": "console.page/route",
     "properties": {
-      "path": "/odf/resource/noobaa.io~v1alpha1~BackingStore/create/~new",
+      "path": "/odf/resource/noobaa.io~v1alpha1~NamespaceStore/create/~new",
       "exact": true,
       "component": {
-        "$codeRef": "bsCreate.default"
+        "$codeRef": "nssCreate.default"
       }
     }
   },
   {
     "type": "console.page/route",
     "properties": {
-      "path": "/odf/resource/noobaa.io~v1alpha1~NamespaceStore/create/~new",
+      "path": "/odf/resource/noobaa.io~v1alpha1~BucketClass/create/~new",
       "exact": true,
       "component": {
-        "$codeRef": "nssCreate.default"
+        "$codeRef": "bcCreate.default"
       }
     }
   },
   {
     "type": "console.page/route",
     "properties": {
-      "path": "/odf/resource/noobaa.io~v1alpha1~BucketClass/create/~new",
-      "exact": true,
+      "path": "/odf/resource/:resourceKind/:resourceName",
+      "exact": false,
```

Comment:
I think, it will affect.
This BZ will work fine even if we do not change the order. But, it will create a new bug instead.
Let's suppose I clicked on the "Create Backing Store" button on the list page. It will check all the available Routes. If this Route (with `exact = false`) is defined before the Route for creating backing-store component then this Route will be matched (because we are using `exact = false` here) and it will not load create-backing-store specific component.